crates/rpc/rpc/src/debug.rs

```diff
@@ -1143,8 +1143,30 @@ where
         Ok(())
     }
 
-    async fn debug_db_get(&self, _key: String) -> RpcResult<()> {
-        Ok(())
+    /// `debug_db_get` - database key lookup
+    ///
+    /// Currently supported:
+    /// - Contract bytecode associated with a code hash.
+    ///   The code hash must be provided as a hex string prefixed with 0x.
```

Comment:
looking at the kona code and how this is parsed

https://github.com/op-rs/kona/blob/aa7e89596afa96f08eb1b2e14db269fabeb673ea/bin/host/src/single/handler.rs#L245

I'm not sure this is correct because if the prefix is missing the this will just use the content

https://github.com/ethereum/go-ethereum/blob/7c107c2691fa66a1da60e2b95f5946c3a3921b00/common/bytes.go#L101-L103